doc/releases/changelog-dev.md

```diff
@@ -2,44 +2,89 @@
 
 <h3>New features since last release</h3>
 
-* Users can now set precisions for a larger variety of `ResourceOperator`s in
-  :mod:`estimator <pennylane.estimator>` using
-  :meth:`ResourceConfig.set_precision <pennylane.estimator.resource_config.ResourceConfig.set_precision>`
-  thanks to the addition of the `resource_key` keyword argument.
-  [(#8561)](https://github.com/PennyLaneAI/pennylane/pull/8561)
+<h4>Bucket-Brigade QRAM </h4>
```

Comment:
Has the feature not been merged yet? 🤔

Reply:
just checked that it was merged, I couldn't find the changelog entry though. I am double checking, but I'll add it if indeed it is missing